### PR TITLE
Fixed the RTL alignment on Hour of Code overview

### PR DESCRIPTION
**Pull request by aaka3207**
Initial fix as part of FND-1482 to fix all text and box alignments for languages in RTL format. using the **language_dir_class** found in pegasus/helpers.rb to add the "dir" attribute to several UI components. 

I attempted to minimize the number of times this function is called by wrapping some blocks of text in a _div_, if there's a more efficient way to handle this I'm open to suggestions. 

### Before
image
image

### After
image
image

(if it looks like it's cut off halfway through it's because each image was cut in half and for some reason I couldn't get a screenshot at each cutoff point)

## Follow-up work

1. **FND-1482** also includes fixing the tutorial overview pages, and other pages linking from the Hour of Code page. This is just to get some initial feedback for this work moving forward. My thinking is it's better to merge in page-by-page instead of waiting for all pages to be fixed.
2. While working on this, I discover that some RTL languages (tested with Arabic) have issues rendering the markdown text in the subheader of this page:
image

I can include a fix to this in a separate PR, or if it's preferred to keep all work for a page in one PR include it here.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/tools.css [hunk 1]
```diff
@@ -78,12 +78,16 @@ h3.tutorial-info-h {
 .tutorial-info-start {
   margin-top: 20px;
 }
-
+.tutorial-info-start[dir='rtl']{
+  float:right !important;
```
Comment (daynew) [inline pegasus/sites.v3/code.org/public/css/tools.css]: We try to avoid using `!important` as much as possible. Was this necessary?

**Review by daynew — CHANGES_REQUESTED** [review]
Is it possible minimize the changes by setting the container of the page content to have `<...container_div... dir="rtl">` instead of each text element in the page? I tried adding `dir="rtl"` to `<div class="container_responsive">` and got the result in the screenshot below:

image

I'm interested to know if this breaks something else or doesn't give us the result we want.

File: pegasus/sites.v3/code.org/public/css/tools.css (hunk 1 above)
Comment (aaka3207) [reply pegasus/sites.v3/code.org/public/css/tools.css]: Actually I just tried again without `!imporant` and it seems to be working just fine, removing.

**Comment by aaka3207** [comment]
> Is it possible minimize the changes by setting the container of the page content to have `<...container_div... dir="rtl">` instead of each text element in the page? I tried adding `dir="rtl"` to `<div class="container_responsive">` and got the result in the screenshot below:
> 
> image
> 
> I'm interested to know if this breaks something else or doesn't give us the result we want.

@daynew So the reason that I didn't change it for the entire container was because it messed with some of the children classes. For example, You can see that the tutorial boxes lose their margins. However, I'm going to change my approach slightly and apply this to all the pages that need RTL and then fix those issues. I'll update this PR and add `dir="rtl"` to that **container_responsive** div.

File: pegasus/sites.v3/code.org/public/css/tools.css (hunk 1 above)
Comment (aaka3207) [reply pegasus/sites.v3/code.org/public/css/tools.css]: I removed them.

**Comment by aaka3207** [comment]
tagging @dju90

**Comment by dju90** [comment]
I think the markdown error is actually just a translation error to the string: https://crowdin.com/translate/codeorg/56/enus-ar#2364764. I've fixed it.

**Review by daynew — APPROVED** [review]
Looks good! I'm happy to see it turned out to be a small change :)

**Pull request merged**